sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -44,6 +44,39 @@ extern __DPCPP_SYCL_EXTERNAL void __spirv_JointMatrixStoreINTEL(
     std::size_t Stride, __spv::MatrixLayout Layout = L,
     __spv::Scope::Flag Sc = S, int MemOperand = 0);
 
+template <typename T, typename Tp, std::size_t R, std::size_t C,
+          __spv::MatrixUse U,
+          __spv::MatrixLayout L = __spv::MatrixLayout::RowMajor,
+          __spv::Scope::Flag S = __spv::Scope::Flag::Subgroup>
+extern __DPCPP_SYCL_EXTERNAL
+    __spv::__spirv_JointMatrixINTEL<Tp, R, C, L, S, U> *
+    __spirv_CompositeConstructCheckedINTEL(const T v, size_t Height,
+                                           size_t stride, size_t Width,
+                                           size_t coordX, size_t coordY);
```

Comment:
The argument names are inconsistent.
```suggestion
    __spirv_CompositeConstructCheckedINTEL(const T Value, size_t Height,
                                           size_t Stride, size_t Width,
                                           size_t CoordX, size_t CoordY);
```
Similar applies to the other declarations and definitions in these changes..